packages/cli/test/unit/util/get-latest-version.test.ts

```diff
@@ -130,6 +130,42 @@ describe('get latest version', () => {
     );
     expect(() => getLatestVersion({ pkg: { name: '' } })).toThrow(TypeError);
   });
+
+  it('should reset notify if newer version is available', async () => {
+    // 1. seed the cache file with both a expireAt and notifyAt in the future
+    //    with an out-of-date latest version
+    await fs.mkdirs(join(cacheDir, 'package-updates'));
+    await fs.writeJSON(cacheFile, {
+      expireAt: Date.now(),
+      notifyAt: Date.now() - 60000,
+      version: '28.0.0',
+    });
+
+    // 2. get the latest version
+    let latest = getLatestVersion({
+      cacheDir,
+      pkg,
+    });
+    expect(latest).toEqual('28.0.0');
+
+    // we need to wait up to 20 seconds for the cacheFile to be updated
+    for (let i = 0; i < 80; i++) {
+      await sleep(250);
```

Comment:
**suggestion (non-blocking):** reduce sleep

For this to work, we only need to relinquish control flow, right? Can we `sleep(0)` instead?

Reply:
This isn't kernel level `sleep()`. This is the `sleep()` function in the `cli/src/util/sleep.ts` file and it's just a promise with a `setTimeout()`. During the `sleep()`, it's making an `fetch()` which is known to take up to 10 seconds in CI.